MyString.java

```diff
@@ -49,8 +68,17 @@ public static boolean subsetOf(String str1, String str2) {
      * @return a string consisting of the characters of str, separated by spaces.
      */
     public static String spacedString(String str) {
-        //// Replace the following statement with your code
-        return null;
+        String newStr = "";
+        if (str == newStr) {
+            return " ";
+        }

```

Comment:
== is not consistent with non primitive types.. 
use `str.equals("")` or `str.length() == 0` to compare with empty strings